src/Microsoft.AspNetCore.Http.Features/IResponseCookiesFeature.cs

```diff
@@ -3,8 +3,14 @@
 
 namespace Microsoft.AspNetCore.Http.Features
 {
+    /// <summary>
+    /// Feature containing cookies which will be returned with the response.
+    /// </summary>
     public interface IResponseCookiesFeature
     {
+        /// <summary>
+        /// Cookies which will be returned with the response.
```

Comment:
Yah want it described as an ease-of-use façade on top of `IHttpResponseFeature`?